articles/azure/azs-how-resize-disk-ps.md

```diff
@@ -208,7 +208,8 @@ Enter details below to provide values for the variables in the following scripts
 
     ```powershell
     # Obtain public IP of your VM based on the variables from above
-    $IpAddress= (Get-AzureRmPublicIpAddress -ResourceGroupName $RGName | Where-Object -FilterScript { $_.IpConfiguration.Id -like "*$VMName*" })
```

Comment:
It seems that the IpConfiguration.Id variable does not contain the name of the virtual machine. Therefore, I've updated this statement to first obtain the network interface which can then be used to obtain the public IP.

Reply:
Makes sense :-)